astropy/tests/pytest_plugins.py

```diff
@@ -147,8 +147,9 @@ def pytest_configure(config):
                           allow_astropy_data=config.getoption('remote_data') == 'astropy')
 
     doctest_plugin = config.pluginmanager.getplugin('doctest')
-    if (doctest_plugin is None or config.option.doctestmodules or not
-            (config.getini('doctest_plus') or config.option.doctest_plus)):
+    if (doctest_plugin is None or config.option.doctestmodules
+            or not (config.getini('doctest_plus').lower() in ['true', 'enabled']
```

Comment:
Nitpick: `flake8` complains about operator not at the end of line, which I happen to agree with, as putting them before line break makes it clear that the condition has something to follow in the next line.